.github/workflows/tests-fetchers.yml

```diff
@@ -0,0 +1,53 @@
+name: Fetcher Tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'src/main/java/org/jabref/logic/importer/fetcher/**'
```

Comment:
Maybe its better to be more general and include `src/main/java/org/jabref/logic/importer` as the parsers are also essential for the fetcher and are a potential source why a fetcher test may break.